src/imitation/scripts/config/parallel.py

```diff
@@ -92,7 +116,7 @@ def example_rl_easy():
 @parallel_ex.named_config
 def example_gail_easy():
   inner_experiment_name = "train_adversarial"
-  outer_experiment_name = "example-gail-easy"
+  inner_run_name = "example-gail-easy"
```

Comment:
Is there anything that is `outer` now? If not, perhaps drop the `inner_` prefix here?

Reply:
I'm renaming `inner_experiment_name` to `sacred_ex_name` and `inner_run_name` => `run_name`.